### ISSUE TITLE
CF orchestrator fails to install as of Sept 2021 - downloading .NET 4.7.2 framework

### ISSUE THREAD

**Issue by obriensystems** [aws-quickstart/quickstart-uipath-orchestrator#30]
see line
https://github.com/aws-quickstart/quickstart-uipath-orchestrator/blob/main/templates/server.template.yaml#L579

getting 
2021-09-12 03:32:32,624 [DEBUG] Response: 404 https://download.microsoft.com/download/6/E/4/6E48E8AB-DC00-419E-9704-06DD46E5F81D/NDP472-KB4054530-x86-x64-AllOS-ENU.exe [headers: {'content-type': 'text/html', 'content-length': '1213', 'date': 'Sun, 12 Sep 2021 03:32:32 GMT', 'connection': 'keep-alive'}]
2021-09-12 03:32:32,624 [DEBUG] Response error: b'<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">\n<html xmlns="http://www.w3.org/1999/xhtml">\n<head>\n<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>\n<title>404 - File or directory not found.</title>\n<style type="text/css">\n\n</style>\n</head>\n<body>\n<div id="header"><h1>404 Not Found</h1></div>\n<div id="content">\n <div class="content-container"><fieldset>\n  <h2>404 - File or directory not found.</h2>\n  <h3>The resource you are looking for might have been removed, had its name changed, or is temporarily unavailable.</h3>\n </fieldset></div>\n</div>\n</body>\n</html>'

attempting workaround switching to
https://dotnet.microsoft.com/download/dotnet-framework/net472
https://dotnet.microsoft.com/download/dotnet-framework/thank-you/net472-web-installer
or
https://go.microsoft.com/fwlink/?LinkID=863265
302 to
https://download.visualstudio.microsoft.com/download/pr/1f5af042-d0e4-4002-9c59-9ba66bcf15f6/089f837de42708daacaae7c04b7494db/ndp472-kb4054530-x86-x64-allos-enu.exe

**Comment by obriensystems** [comment]
found via

attempting workaround switching to
https://dotnet.microsoft.com/download/dotnet-framework/net472 
https://dotnet.microsoft.com/download/dotnet-framework/thank-you/net472-web-installer 
or
https://go.microsoft.com/fwlink/?LinkID=863265 
302 to
https://download.visualstudio.microsoft.com/download/pr/1f5af042-d0e4-4002-9c59-9ba66bcf15f6/089f837de42708daacaae7c04b7494db/ndp472-kb4054530-x86-x64-allos-enu.exe 



diff --git a/cloudformation/secure-templates-stg4-full-gctip/server.template.yaml b/cloudformation/secure-templates-stg4-full-gctip/server.template.yaml
index 1c23653..3e7fd36 100644
--- a/cloudformation/secure-templates-stg4-full-gctip/server.template.yaml
+++ b/cloudformation/secure-templates-stg4-full-gctip/server.template.yaml
@@ -530,15 +530,16 @@ Resources:
           files:
             c:\cfn\sources\UiPathOrchestrator.msi:
               source: !Sub 'https://download.uipath.com/versions/${OrchestratorVersion}/UiPathOrchestrator.msi'
-            c:\cfn\sources\NDP472-KB4054530-x86-x64-AllOS-ENU.exe:
-              source: https://download.microsoft.com/download/6/E/4/6E48E8AB-DC00-419E-9704-06DD46E5F81D/NDP472-KB4054530-x86-x64-AllOS-ENU.exe
+            c:\cfn\sources\ndp472-kb4054530-x86-x64-allos-enu.exe:
+              #source: https://download.microsoft.com/download/6/E/4/6E48E8AB-DC00-419E-9704-06DD46E5F81D/NDP472-KB4054530-x86-x64-AllOS-ENU.exe
+              source: https://download.visualstudio.microsoft.com/download/pr/1f5af042-d0e4-4002-9c59-9ba66bcf15f6/089f837de42708daacaae7c04b7494db/ndp472-kb4054530-x86-x64-allos-enu.exe
             c:\cfn\sources\rewrite_amd64_en-US.msi:
               source: https://download.microsoft.com/download/1/2/8/128E2E22-C1B9-44A4-BE2A-5859ED1D4592/rewrite_amd64_en-US.msi
             c:\cfn\sources\dotnet-hosting-3.1.8-win.exe:
               source: https://download.visualstudio.microsoft.com/download/pr/854cbd11-4b96-4a44-9664-b95991c0c4f7/8ec4944a5bd770faba2f769e647b1e6e/dotnet-hosting-3.1.8-win.exe
           commands:
             a-install-dotnet-framework:
-              command: start /wait c:\cfn\sources\NDP472-KB4054530-x86-x64-AllOS-ENU.exe /q /norestart
+              command: start /wait c:\cfn\sources\ndp472-kb4054530-x86-x64-allos-enu.exe /q /norestart
               waitAfterCompletion: '0'
             b-install-roles-and-features:
               command: powershell.exe -command C:\cfn\scripts\Install-RolesAndFeatures.ps1
diff --git a/cloudformation/secure-templates-stg4-full-gctip/uipath-orchestrator.template.yaml b/cloudformation/secure-templates-stg4-full-gctip/uipath-orchestrator.template.yaml
index 6b3311a..9cc7025 100644
--- a/cloudformation/secure-templates-stg4-full-gctip/uipath-orchestrator.template.yaml
+++ b/cloudformation/secure-templates-stg4-full-gctip/uipath-orchestrator.template.yaml
@@ -836,9 +836,9 @@ Parameters:
   OrchestratorVersion:
     Description: UIPath Orchestrator version.
     Type: String
-    Default: 20.10.7
+    Default: 21.4.0
     AllowedValues:
-      - 20.10.7
+      - 21.4.0
     ConstraintDescription: UIPath Orchestrator version can only contain characters *0–9* and *.* (period).





after this change the public install came up

private AWS account
uipath115-pub-OrchestratorStack-41Z4JV56U12C-ServerStack-CCLTB0U5D57T
NESTED
CREATE_COMPLETE 2021-09-12 00:59:10 UTC-0400 This template creates a cluster to provide high availability.

**Comment by obriensystems** [comment]
![image](https://user-images.githubusercontent.com/24765473/132988374-4ca15cc3-62be-4b45-94b4-536a7fdebac7.png)

**Comment by ubikusss** [comment]
Yes, thank you very much, we will change the URL in our next PR.
